src/selfie_intersection/src/IntersectionServer.cpp

```diff
@@ -0,0 +1,104 @@
+/**
```

Comment:
W zasadach pisania kodu umawialiśmy się na `takie_nazwy_plików.cpp`

Reply:
Racja (zapomniałem, a ponieważ w szukaniu miejsca też tak ponazywałem to tamto też będzie trzeba poprawić)